packages/react-native/src/modules/feed/components/NotificationFeedComponents/NotificationFeedCell/NotificationFeedCell.tsx

```diff
@@ -0,0 +1,246 @@
+          }),
+        sentAtDateTextStyle: styleOverride?.sentAtDateTextStyle ?? {
+          fontSize: theme.fontSizes.knock2,
+          color: theme.colors.gray9,
+          fontWeight: theme.fontWeights.medium,
+        },
+        htmlStyles: styleOverride?.htmlStyles ?? {
+          p: {
+            fontSize: theme.fontSizes.knock2,
+            fontFamily: theme.fontFamily.sanserif,
+            margin: 0,
+            color: theme.colors.gray12,
+          },
+          blockquote: {
+            color: theme.colors.gray11,
+            borderLeftWidth: 4,
+            borderLeftColor: theme.colors.gray6,
+            paddingLeft: 12,
+            paddingVertical: 4,
+            margin: 0,
+            marginTop: 12,
+          },
+        },
+      }),
+      [theme, styleOverride],
+    );
+
+    const { width } = useWindowDimensions();
+
+    const renderMarkdownContent = useCallback(
+      (block: MarkdownContentBlock) => (
+        <RenderHTML
+          contentWidth={width}
+          source={{ html: block.rendered }}
+          tagsStyles={resolvedStyle.htmlStyles as any}
+        />
+      ),
+      [width, resolvedStyle.htmlStyles],
+    );
+
+    const renderActionButtonsContent = useCallback(
+      (block: ButtonSetContentBlock) => (
+        <View style={layoutStyles.buttonContainer}>
+          {block.buttons.map((button, index) => (
+            <ActionButton
+              key={`button-${index}`}
+              title={button.label}
+              type={
+                button.name === "primary"
+                  ? ActionButtonType.PRIMARY
+                  : ActionButtonType.SECONDARY
+              }
+              styleOverride={
+                button.name === "primary"
+                  ? resolvedStyle.primaryActionButtonStyle
+                  : resolvedStyle.secondaryActionButtonStyle
+              }
+              action={() => onCellActionButtonTap({ button, item })}
+            />
+          ))}
+        </View>
+      ),
+      [onCellActionButtonTap, item],
+    );
+
+    return (
+      <TouchableOpacity onPress={() => onRowTap(item)}>
+        <View style={layoutStyles.container}>
+          <View style={layoutStyles.row}>
+            <View style={layoutStyles.leftContainer}>
+              <View
+                style={[
+                  layoutStyles.circle,
+                  isRead
+                    ? { backgroundColor: "transparent" }
+                    : {
+                        backgroundColor:
+                          resolvedStyle.unreadNotificationCircleColor,
+                      },
+                ]}
+              />
+              {resolvedStyle.showAvatarView && actor && isUser(actor) && (
+                <View>
+                  <AvatarView name={actor.name} src={actor.avatar} />
+                </View>
+              )}
+            </View>
+            <View style={layoutStyles.contentContainer}>
+              {item.blocks.map((block, index) => {
```

Comment:
I think one thing we're missing here right now is support for the `action_url` block, which is a text block that contains the action that should be invoked on tap of the notification cell.